deploy/manifests/base/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ resources:
 configMapGenerator:
   - name: autoretrieve-config
     files:
-      - config.yaml
+      - config.yaml.erb
```

Comment:
I don't seem to be able to find a file with this name. 
Can you help me get my head around the rationale for this change please?

Reply:
Not sure about initial intent, but removing the `.erb` extension fixed the `kustomize build` error.